### DISCUSSION TITLE
When will RTC mobile be supported?

### DISCUSSION THREAD

**Discussion by dynafer-archive** [tinymce/tinymce#7791] [General]
I have read the documentation that RTC doesn't support mobile browser.

However, when I tried it on iPhone and Galaxy, it worked on iPhone; however, it didn't work on Galaxy.
How it didn't work on Galaxy, I couldn't type letters but deletion of letter, typing of numbers worked. I just need typing letters works properly.

**Comment by lostkeys** [comment]
Hi @dynafer. I'm sorry we are not available to provide you with an answer on when we may support RTC mobile right now.
Fredrik – Product Manager
